xblock/django/request.py

```diff
@@ -146,15 +146,15 @@ def POST(self):
             querydict_to_multidict(self._request.FILES, wrap=DjangoUploadedFile),
         )
 
-    @lazy
-    def body(self):  # pylint: disable=method-hidden
+    @property
```

Comment:
please add a reason for this change.

Reply:
Both of these are marked as `property` in class we are inheriting from so pylint suggested making it `property` in child class too.